rust/operator-binary/src/security/authentication.rs

```diff
@@ -113,20 +130,32 @@ impl NifiAuthenticationConfig {
     pub fn get_additional_container_args(&self) -> Vec<String> {
         let mut commands = Vec::new();
         match &self {
-            Self::SingleUser(_) => {
+            Self::SingleUser { .. } => {
                 let (_, admin_password_file) = self.get_user_and_password_file_paths();
                 commands.extend(vec![
                     format!("export STACKABLE_ADMIN_PASSWORD=\"$(cat {admin_password_file} | java -jar /bin/stackable-bcrypt.jar)\""),
                 ]);
             }
-            Self::Ldap(ldap) => {
-                if let Some(ca_path) = ldap.tls.tls_ca_cert_mount_path() {
+            Self::Ldap { provider } => {
+                if let Some(ca_path) = provider.tls.tls_ca_cert_mount_path() {
                     commands.extend(vec![
                         "echo Adding LDAP tls cert to global truststore".to_string(),
                         format!("keytool -importcert -file {ca_path} -keystore {STACKABLE_SERVER_TLS_DIR}/truststore.p12 -storetype pkcs12 -noprompt -alias ldap_ca_cert -storepass {STACKABLE_TLS_STORE_PASSWORD}"),
                     ]);
                 }
             }
+            Self::Oidc { provider, .. } => {
+                let (_, admin_password_file) = self.get_user_and_password_file_paths();
+                commands.extend(vec![
+                format!("export STACKABLE_ADMIN_PASSWORD=\"$(cat {admin_password_file} | java -jar /bin/stackable-bcrypt.jar)\""),
```

Comment:
Ahhhh I would be nice if https://github.com/stackabletech/config-utils/ could do the bcrypt for us :)

Reply:
Totally agree, there's always room for improvements :)